spec/mock/mockWindow.js

```diff
@@ -9,7 +9,7 @@ function mockWindow() {
             protocol: "http:",
             host: "example.com:8080",
             pathname: "/requested/path",
-            search: "?some=param&another[param]=value",
+            search: "?some=param&another%5Bparam%5D=value",
```

Comment:
I don't think this is critical. But we may not want to make the mockWindow to have the search attribute globally. Ideally, it would be nice to do something like
```js
var mockWindow = mockWindow().withSearch("?some=param&another%5Bparam%5D=value");
```